cmd/ddev/cmd/config.go

```diff
@@ -470,16 +465,11 @@ func handleMainConfigArgs(cmd *cobra.Command, _ []string, app *ddevapp.DdevApp)
 		app.HostDBPort = hostDBPortArg
 	}
 
-	if cmd.Flag("nfs-mount-enabled").Changed {
-		app.NFSMountEnabled = nfsMountEnabled
-	}
-
-	if cmd.Flag("mutagen-enabled").Changed {
-		app.MutagenEnabled = mutagenEnabled
-		if app.IsNFSMountEnabled() {
-			util.Warning("nfs-mount-enabled disabled because incompatible with mutagen, which is enabled")
-			app.NFSMountEnabled = false
-			app.NFSMountEnabledGlobal = false
+	if cmd.Flag(types.FlagPerformance).Changed {
+		if err := types.CheckValidPerformance(performance); err != nil {
+			util.Error("%s. Not changing performance.", err)
```

Comment:
The one problem with our naming this "performance" is that people are going to think we're talking about performance itself, rather than about a configuration option. I wonder if that's going to be a problem. @mattstein could you weigh in? 

```suggestion
			util.Error("Error: '%s'. Not changing value of `performance` option.", err)
```